tests/features/steps/steps.py

```diff
@@ -34,6 +35,48 @@ def step_impl_send_client(context, model_name, method_name):
     context.response = method(**kwargs)
     store_last_item(context, model_name)
 
+@when('we upload a binary file with metadata.{model_name}.{method_name}')
```

Comment:
These steps are to imitate how a user would use the client library. Therefor can you change this one, and the step underneath, to the following:
`we upload a binary file with client.{model_name}.{method_name}` and
`we download a binary file with client.{model_name}.{method_name}`

Reply:
Resolved